<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RadioCtl 1.0</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <widget class="QTreeWidget" name="lstDir">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QTabWidget" name="tabView">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionNewDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionTerminal"/>
    <addaction name="actionTop"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionRenameFileDirectory"/>
    <addaction name="actionDeleteFileDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionCloseTab"/>
    <addaction name="actionCloseAll"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Session">
    <property name="title">
     <string>&amp;Session</string>
    </property>
    <addaction name="actionNewSession"/>
    <addaction name="separator"/>
    <addaction name="actionReconnect"/>
    <addaction name="actionQuitSession"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionShowHidden"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_Session"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="barStandard">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewFile"/>
   <addaction name="actionNewDirectory"/>
   <addaction name="separator"/>
   <addaction name="actionOpenFile"/>
   <addaction name="actionTerminal"/>
   <addaction name="actionTop"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionReconnect"/>
   <addaction name="separator"/>
   <addaction name="actionQuitSession"/>
  </widget>
  <widget class="QToolBar" name="barAddress">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionNewFile">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;New File...</string>
   </property>
  </action>
  <action name="actionNewDirectory">
   <property name="icon">
    <iconset theme="folder-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>N&amp;ew Folder...</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionRenameFileDirectory">
   <property name="text">
    <string>Rena&amp;me File/Folder...</string>
   </property>
  </action>
  <action name="actionDeleteFileDirectory">
   <property name="icon">
    <iconset theme="list-remove">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete File/Folder...</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="icon">
    <iconset theme="network-wired">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Re&amp;connect</string>
   </property>
  </action>
  <action name="actionQuitSession">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit Session</string>
   </property>
  </action>
  <action name="actionNewSession">
   <property name="icon">
    <iconset theme="window-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New &amp;Session...</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut File</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy File</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste File</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="text">
    <string>&amp;Close Tab</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>Close A&amp;ll</string>
   </property>
  </action>
  <action name="actionGo">
   <property name="icon">
    <iconset theme="go-next">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="icon">
    <iconset theme="utilities-terminal">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Terminal</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="icon">
    <iconset theme="hwinfo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </action>
  <action name="actionShowHidden">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Hidden Files</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset theme="configure">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="actionCopyText">
   <property name="icon">
    <iconset theme="edit-copy"/>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPasteText">
   <property name="icon">
    <iconset theme="edit-paste"/>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionKeyStroke">
   <property name="icon">
    <iconset theme="preferences-desktop-keyboard"/>
   </property>
   <property name="text">
    <string>Send Key Stroke</string>
   </property>
  </action>
  <action name="actionScrollbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Scrollbar</string>
   </property>
  </action>
  <action name="actionColorScheme">
   <property name="text">
    <string>Set Color Scheme</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset theme="preferences-desktop-font"/>
   </property>
   <property name="text">
    <string>Set Font</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="tab-close"/>
   </property>
   <property name="text">
    <string>Close Tab</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
